test-vectors/rust/issuance_auth_sig.rs

```diff
@@ -1,4 +1,4 @@
-// From https://github.com/zcash-hackworks/zcash-test-vectors/ (zsa_issuance_auth_sig)
+// From https://github.com/QED-it/zcash-test-vectors/ (issuance_auth_sig)
```

Comment:
manual changes in `.rs` files are not going to persist a regeneration, revert this change.

Reply:
Revert to `// From https://github.com/zcash-hackworks/zcash-test-vectors/ (issuance_auth_sig)`